test-suite/swingoption.cpp

```diff
@@ -320,10 +321,11 @@ void SwingOptionTest::testFdBSSwingOption() {
             lowerBound += europeanOption.NPV();
         }
 
-        if (lowerBound - swingOptionPrice > 2e-2) {
+        if (lowerBound - swingOptionPrice > 0.1) {
```

Comment:
Is this a typo, or did the tolerance have to increase 5x?

Reply:
The difference on my Linux box was getting close to the limit with 0.0175 and I've put some extra buffer in for other platforms, but you are right 4e-2 should be enough here.